docs/source/_toctree.yml

```diff
@@ -1,14 +1,16 @@
 - sections:
   - local: index
     title: 🤗 Hugging Face Hub Python Wrapper
+  - local: how-to-manage
+    title: How to manage and create repositories
   - local: how-to-downstream
-    title: How to download files from the hub
+    title: How to download files
```

Comment:
Maybe instead of how to ..., we should change to "Downloading files", "Uploading files", etc.